src/utils/removeTag.ts

```diff
@@ -0,0 +1,11 @@
+export const removeParagraphTag = (inputString: string): string => {
```

Comment:
function also removing other tags, not only paragraph tags. could have been called removeTag